SleepEngine/Engine/Utils/MacroUtils.h

```diff
@@ -124,7 +124,9 @@ inline componentType const& shortcutName(Object const* obj) \
 }   \
     \
 
-
+#define ALL(container) std::begin(container), std::end(container)
+#define C_ALL(container) std::cbegin(container), std::cend(container)
+#define MOVE_ALL(container) std::make_move_iterator(std::begin(container)), std::make_move_iterator(std::end(container))
```

Comment:
not using theese

Reply:
Yeah, I wanted to use this but I found a `std::move_backward` that uses simple iterators. Should I delete it? It possibly may be used in the future.